lib/Context/Internal/ProcessHub.php

```diff
@@ -33,7 +33,7 @@ class ProcessHub
 
     public function __construct()
     {
-        $isWindows = \strncasecmp(\PHP_OS, "WIN", 3) === 0;
+        $isWindows = 'Windows' === \PHP_OS_FAMILY;
```

Comment:
```suggestion
        $isWindows = \PHP_OS_FAMILY === 'Windows';
```

Please avoid Yoda style.